tests/unit/sparql/buildQuery.spec.ts

```diff
@@ -14,6 +14,29 @@ describe( 'buildQuery', () => {
 		} } ) ).toEqual( `SELECT ?item WHERE { ?item (p:${propertyId}/ps:${propertyId}) "${value}". }` );
 	} );
 
+	it( 'builds a query from a property and a string value with not matching selected', () => {
+		const propertyId = 'P666';
+		const value = 'blah';
+		const propertyValueRelation = PropertyValueRelation.NotMatching;
+
+		/*
+		* adding an array here to circumvent backticks + indentation problems between
```

Comment:
I think the more scalable solution might be to just use backticks here and not worry about spaces.

In the assert, we can then replace any whitespace `\s+` with a single space `' '` for _both_ the expected and the actual query. See [String.prototype.replaceAll()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll)

Reply:
the problem is that if you use back ticks then the normal indentation of the code gets added as tabs and it breaks the test.

Reply:
Alternatively we can not indent the code but I don't think that's an ideal solution and it would create lint errors. Also leaving it in online line and adding `\n` makes the line too long. I tried all different ways and this seems the most decent solution.

Reply:
Given that these are tests, I wouldn't worry too much. If it gets really big, you can move it a file and read that instead IMO

Reply:
As I wrote above, we can replace all the whitespace, including any tabs and newlines and such, with a single space before actually asserting the equality. That should ensure that the test still works and still tests that the actual code is equal, right?

Reply:
I tried with trim and it didn't work well, will try with replaceAll

Reply:
for the record, `replaceAll` is not supported by all browsers and causes the tests to fail. did the same with `replace`. https://caniuse.com/?search=replaceAll

Reply:
I'm curious to see how replaceAll failed, but I'm fine with just using replace.

Now all that still needs doing is removing this now obsolete commend and then the PR can be approved and merged :)

Reply:
It says `replaceAll` is not a function.